test/FateRewardControllerV3.js

```diff
@@ -123,7 +123,7 @@ describe('FateRewardControllerV3', () => {
             const receivedAmount = bobAfterLPAmount.sub(bobBeforeLPAmount)
 
             // check received amount after withdraw: withdrawAmount * (100 - lpWithdrawFee)
-            expect(receivedAmount).to.be.equal(withdrawAmount.mul(100 - 88).div(100))
+            expect(receivedAmount).to.be.eq(withdrawAmount.mul(100 - 88).div(100))
```

Comment:
This line is failing. Likely the same issue as the other test down below I commented on.